rero/rero-ils
# Library: date exceptions

* NEW: add date exceptions in the library JSON schema
* NEW: update editor to manage library

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Nicolas Labat <nicolas.labat@rero.ch
Signed-off-by: Peter Weber <Peter.Weber@rero.ch>

How to test:
Run the scripts `bootstrap` and `setup`

The library "Bibliothèque cantonale valdôtaine, site de Pont-Saint-Martin"
has predefined exception dates.  
* check detailed library view
* edit a library and add or change exception dates
